js/gdm/realmd.js

```diff
@@ -68,6 +68,7 @@ var Manager = new Lang.Class({
                                            '/org/freedesktop/realmd',
                                            Lang.bind(this, this._reloadRealms))
         this._realms = {};
+        this._loginFormat = null;
```

Comment:
There’s a check for `(this._loginFormat !== undefined)` on line 138. Will this change break that?

Reply:
I haven't observed any issue with this change but this is a fairly good point. I will double check the need of this patch then once again, since it is possible this was hit by an earlier version of my branch and maybe now it's not needed. But if it is, I agree I would probably need to be more careful since that can potentially break things.

Reply:
Actually, this patch **is not needed anymore** as I confirmed by reverting it and running my local tests once again. Removing it

Reply:
Wheeee

Reply:
The patch is still here in the branch on github, but presumably you’ve dropped it locally. This all looks good for you to merge.